test/unit/sth_test.js

```diff
@@ -163,6 +163,8 @@ describe('sth tests', function() {
 
     it('should respond with 200 - OK if lastN query param', sthTestUtils.status200Test.bind(null, {lastN: 1}));
 
+    it('should respond with 200 - OK if count query param', sthTestUtils.status200Test.bind(null, {lastN: 1, count: 1}));
```

Comment:
I guess that `count=true` (at API level, as decribed in documentation) translates to `count: 1` in this line of test code. However, should the test check that the fiware-total-count header is included in the response in some way (I'm not fully sure... I lack of context of the tests)?